apps/frontend/src/pages/community/index.tsx

```diff
@@ -533,10 +561,43 @@ export default function CommunityPage() {
             </div>
             <div className="flex flex-col gap-2">
               <span className="text-base font-bold text-primary font-sans">
-                Coming soon!
+                Coming soon
               </span>
               <ComingSoonCommunityGoals />
             </div>
+            <div className="flex flex-col gap-2">
+              <span className="text-base font-bold text-primary font-sans">
+                {`Past dashboards`}
```

Comment:
I like this feature, I'm a sucker for digging into historical patterns / trends.